roles/infrastructure/tasks/rabbitmq_update.yml

```diff
@@ -0,0 +1,96 @@
+- name: Atualização do RabbitMQ
+  block:
+  # parar containers viasoft.* e korp.* (validar que isso inclui todos)
+  - name: "Obter containeres korp e viasoft"
+    ansible.builtin.shell:
+      docker ps -q
+    register: active_services_id
+
+  - name: "Obter containeres korp e viasoft"
+    include_tasks: get_service_id_from_image_name.yml
+    loop: "{{ active_services_id.stdout_lines }}"
+
+  - name: Obter id de container do RabbitMQ
+    ansible.builtin.shell:
+      docker ps -q --filter name={{ rabbitmq_container_name }}
+    register: rabbitmq_container_output
+
+  - name: Definição de id de container do RabbitMQ
+    ansible.builtin.set_fact:
+      rabbitmq_container_id: "{{ rabbitmq_container_output.stdout }}"
+
+  - name: Redefinição de 'services_to_restart'
+    ansible.builtin.set_fact:
+      services_to_restart: "{{ services_to_restart | default([]) | difference([rabbitmq_container_id]) }}"
+
+  - name: Parada de containeres
+    ansible.builtin.shell:
+      docker stop {{ item }}
+    loop: "{{ services_to_restart }}"
+
+  - name: Parada de container do RabbitMQ desatualizado
+    ansible.builtin.shell:
+      docker stop {{ rabbitmq_container_id }}
+    when: rabbitmq_container_id != ""
+
+  - name: Backup de volume do RabbitMQ
+    ansible.builtin.copy:
+      src: "{{ dados_docker_dir_path }}/{{ rabbitmq_container_name }}/" 
+      dest: "{{ dados_docker_dir_path }}/rabbitmq_backup/"
+      owner: "{{ linux_korp.user }}"
+    become: true
+
+  - name: Remoção de container do RabbitMQ desatualizado
+    ansible.builtin.shell:
+      docker rm -f {{ rabbitmq_container_id }}
+    when: rabbitmq_container_id != ""
+
+  - name: Atualização progressiva
+    ansible.builtin.include_tasks: rabbitmq_update_to_version.yml
+    loop:
+      - '3.9'
+      - '3.10'
+      - '3.11'
+      - '3.12'
+      - '3.13'
+
+  - name: Remoção de diretório para substituição
+    ansible.builtin.file:
+      path: "{{ dados_docker_dir_path }}/{{ rabbitmq_container_name }}_3-13/"
+      state: absent
+    become: true
```

Comment:
Agora está criando a pasta com sufixo _3-13/ vazia no início do setup por padrão; Preciso fazer isso pra próxima task não dar erro.

Reply:
muito arriscado fazer isso assim. Cria um task garantindo que a pasta está vazia antes de remover ela